4-tools/manual.ipynb

```diff
@@ -521,11 +527,9 @@
     "\n",
     "Now you can see the slide type where <strong style=\"color:red\">b</strong> is in the above image. Every cell starting with `Slide` will be a new slide. Try out the other options to quickly figure out what they do.\n",
     "\n",
-    "## Version control\n",
-    "\n",
-    "It is a good idea to have some version control software manage your code. Not only would this allow you to restore older versions of your code repository, it can also help you to document how the code has evolved. If you are using [Git](https://git-scm.com) it is very simple to host your code (either privately or publicly) on [GitHub](https://github.com), [GitLab](https://gitlab.com) or [Bitbucket](https://bitbucket.org/) (this list is far from being complete). This functions both as a backup in the cloud but also allows you to easily share your code with your collaborators (or at the very least your supervisor), though Git can certainly be useful even if you never share your repository with anyone.\n",
+    "## Scientific Python\n",
     "\n",
-    "If you are interested in version control then you can read more about it from [Chapter 1 Section 1 of the Pro Git book](https://git-scm.com/book/en/v2/Getting-Started-About-Version-Control). If you are interested in using Git then you can continue on reading the book."
+    "The [scientific Python community](https://learn.scientific-python.org/development/) has tonnes of useful materials that can be greatly beneficial to advanced students. In particular, they have guidance on the [process](https://learn.scientific-python.org/development/principles/process/) (how coding projects should be conducted) and [design](https://learn.scientific-python.org/development/principles/design/) (how code should be written), as well as more in-depth guides on topics already covered like [style](https://learn.scientific-python.org/development/guides/style/), [testing](https://learn.scientific-python.org/development/guides/pytest/), and [writing documentation](https://learn.scientific-python.org/development/guides/docs/) and [much more](https://learn.scientific-python.org/development/guides/)."
```

Comment:
Good to have these links, but our Python course does not cover writing documentation.

Reply:
Fair point. I will remove that link.